R/grade_learnr.R

```diff
@@ -1,16 +1,14 @@
-#' grade_learnr
+#' A checker function to use with \code{learnr}.
 #'
-#' A checker function to use with learnr
-#'
-#' For exercise checking, learnr tutorials require a function that learnr can
+#' For exercise checking, \code{learnr} tutorials require a function that \code{learnr} can
 #' use in the background to run the code in each "-check" chunk and to format
-#' the results into a format that learnr can display. The function must accept a
+#' the results into a format that \code{learnr} can display. The function must accept a
 #' specific set of inputs and return a specific type of output (see \code{\link{graded}}).
-#' Users are not intended to use the function themselves, but to pass it to the
+#' Instructors are not intended to use the function themselves, but to pass it to the
 #' \code{exercise.checker} knitr chunk option within the setup chunk of the
-#' tutorial.
+#' \code{learnr} tutorial.
 #'
-#' The grader package provides \code{grade_learnr} for this purpose. To enable
+#' The \code{gradethis} package provides \code{\link{grade_learnr}} for this purpose. To enable
```

Comment:
this first sentence seems like a repeat of the prior paragraph. Remove it? (start with `To enable...`)